server/request.js

```diff
@@ -7,6 +7,7 @@ let cookieParser = require("cookie-parser");
 let cookieSession = require("cookie-session");
 
 let version = require("../package").version;
+let commit = require("../node_modules/git-rev-sync/").long();
```

Comment:
You don't need to use `../node_modules/`, just do:

```js
let gitRevSync = require("git-rev-sync").long();
```

Reply:
+1 to not using "node_modules" in the path. But let's call `long()` only once.
So, it should look like: 
```js
const commit = require("git-rev-sync").long();
...
res.json({ http: "okay", version, commit });
```